src/AmbientSounds.Uwp/MultilingualResources/AmbientSounds.Uwp.zh-TW.xlf

```diff
@@ -0,0 +1,997 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
+  <file datatype="xml" source-language="en-US" target-language="zh-CN" original="AMBIENTSOUNDS.UWP/STRINGS/EN-US/RESOURCES.RESW" tool-id="MultilingualAppToolkit" product-name="n/a" product-version="n/a" build-num="n/a">
```

Comment:
`zh-CN` needs to be changed I think?

Reply:
Right, I just copied the file from zh-CN obviously XD

Reply:
Updated. Did a file search of `CN` and found nothing else.